ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/config/DiscoveryConfiguration.java

```diff
@@ -27,6 +27,13 @@
 import java.util.stream.Stream;
 
 public class DiscoveryConfiguration {
+  public static String GOERLI_DISCOVERY_URL =
+      "enrtree://AKA3AM6LPBYEUDMVNU3BSVQJ5AD45Y7YPOHJLEF6W26QOE4VTUDPE@all.goerli.ethdisco.net";
```

Comment:
Should these three constants be declared `final`?

Reply:
added finals